src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the last item in a list should not have a trailing comma or in this case a margin to the right. It's better to use `:not(:last-child)` to add margin only to the necessary elements.